docs/tutorials/tutorials_intro.md

```diff
@@ -1,3 +1,19 @@
 ## Arm Mbed tutorials
 
-The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides step-by-step instructions for specific tasks you may wish to perform. These include Blinky, our example application; applications that build on Blinky; and debugging tasks. There are also tutorials about using the Arm Mbed Online Compiler, optimizing memory and optimizing power.
+The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides step-by-step instructions for specific tasks you may wish to perform. These include our quick start guide, example applications that build on it and debugging tasks. There are also tutorials about using the Arm Mbed Online Compiler, optimizing memory and optimizing power.
+
+The [Mbed OS quick start](mbed-os-quick-start.html) is our quick start guide, a tutorial that shows you how to set up and run an example application.
+
+[Major releases](major-releases.html) discusses the difference between Mbed OS 2 and Mbed OS 5 and answers frequently asked questions about using the two major versions of Mbed OS. This section also teaches you how to migrate projects to Mbed OS 5.
+
+[Using the Mbed Online Compiler](using-the-mbed-online-compiler.html) uses screenshots and detailed instructions to teach you how to use our online IDE. It guides you through the process of creating, developing and publishing a process and includes advanced steps, such as collaborating with multiple authors.
```

Comment:
"the process... and publishing a process"?

Reply:
That should be program. Good catch.